controller/base-controller.js

```diff
@@ -162,6 +164,25 @@ module.exports = class BaseController extends EventEmitter {
     return validator(key, req.form.values[key], req.form.values, emptyValue);
   }
 
+  _sanitize(req, res, callback) {
+    // If we don't have any data, no need to progress
+    if(req.form.values) {
```

Comment:
Also a test to check for empty req.form.values in case this is removed too. This is always an array right?

Reply:
It's an object not an array so I will check for an empty object (ie. {}) as well as undefined.

Reply:
Maybe:
```
if(!_.isEmpty(req.form.values)) {
```

Reply:
Then add a test for `req.form.values = {}` and make sure the unit tests do as they should.